Week-6/Exercise6_1.js

```diff
@@ -0,0 +1,17 @@
+function sumMax(arr,n){
+    var currentMax = arr[0];
+    var maxSoFar = arr[0];
+
+    for(var i=1 ; i<n ; i++){
```

Comment:
let's assume I don't understand programming, explain me this with layman terms, what and why are you doing here in loop?